### DISCUSSION TITLE
Which is the cubic Crystalshape

### DISCUSSION THREAD

**Discussion by apulgarinc** [mtex-toolbox/mtex#1306] [Ask Anything] [answered]
I want to use the cirstal shape os a Iron-Alpha and i dont know what to use:
I want to do something similar to this but with a cubic crystal shape
% define the crystal shape of Forsterite and store it in the variable cS
cS = crystalShape.olivine(ebsd('Forsterite').CS)

% select only grains with more then 100 pixels
grains = grains(grains.grainSize > 100);

% plot crystal shapes at the positions of the Forsterite grains
hold on
plot(grains('Forsterite'),0.7*cS,'FaceColor',[0.3 0.5 0.3])
hold off

**Comment by kilir** [comment, marked as answer]
Hi,
```
cS = crystalShape.cube(CS)
```
See:
https://mtex-toolbox.github.io/CrystalShapes.html

Cheers,
Rüdiger
